scripts/lstchain_data_muon_analysis.py

```diff
@@ -150,9 +150,13 @@
 
         print("Event {}. Number of pixels above 10 phe: {}".format(event_id,
                                                                   np.size(image[0][image[0] > 10.])))
-        if((np.size(image[0][image[0]>10.]) > 300) or (np.size(image[0][image[0]>10.]) < 50)):
-            continue
+        
+        #if not tag_pix_thr(image): #default skipps pedestal and calibration events
```

Comment:
at the moments no events are tagged with these lines commented out

Reply:
That was because with default values no tagging is applied. I put the thresholds I found in the analysis presented at the INFN MC meeting, but they should be updated once the calibration is alright. 
Regarding your request to include a tagging with the nr of pixels above a given pe thr, it is not so performant as the previous one, so I added a more generic function, (not specifically for muon tagging), that can be used to substitute these lines
if((np.size(image[0][image[0]>10.]) > 300) or (np.size(image[0][image[0]>10.]) < 50)):
            continue
Since using the image size-based tagging makes those lines redundant I commented them out.

Reply:
Now The script should analyze only events passing the image-size double thr conditions according to the values I found on run 442.

Reply:
I'm not sure what do you mean by more performant, at the moment we would like to store images of events that may be coming from muons (or not), without applying any cuts assuming their size, width, radius, etc...
The filter that was applied (actually, `muon_filter` or something like that would probably be more appropriate naming) was assuring to get some pixels above threshold (>50), but getting rid of calibration events (<300, actually I think that this number could also be increased). For those images we would be performing a fit and storing values in the final table. I also added a tag that was evaluating whether the muon is good or not under standard analysis conditions, but this will not be the case until the telescope is in a good shape. Until then, we need a much wider range of events to pass the cuts, and therefore the cut in `size` should be introduced but not used until a later stage.

Reply:
okay, thanks for the clarifications. So I will do the following changes:

1. change tag_muon -> muon_filter
2. leave default values i.e. no filtering is applied
3. uncomment lines skipping calibration and pedestal events leaving default threshold (50 and 300)

Reply:
> I'm not sure what do you mean by more performant, at the moment we would like to store images of events that may be coming from muons (or not), without applying any cuts assuming their size, width, radius, etc...
a selection based on what you suggested earlier, would eliminate much more good muon candidates with respect to the selection in size

> The filter that was applied (actually, `muon_filter` or something like that would probably be more appropriate naming) was assuring to get some pixels above threshold (>50), but getting rid of calibration events (<300, actually I think that this number could also be increased). For those images we would be performing a fit and storing values in the final table. I also added a tag that was evaluating whether the muon is good or not under standard analysis conditions, but this will not be the case until the telescope is in a good shape. Until then, we need a much wider range of events to pass the cuts, and therefore the cut in `size` should be introduced but not used until a later stage.